components/pricing_modal/content.tsx

```diff
@@ -177,7 +177,8 @@ function Content(props: ContentProps) {
     }
 
     const openPurchaseModal = () => {
-        trackEvent('cloud_admin', 'click_open_purchase_modal');
+        const utcTimeStamp = new Date().toISOString();
+        trackEvent('cloud_admin', 'click_open_purchase_modal', {utcTimeStamp, epochTimeStamp: Date.parse(utcTimeStamp)});
```

Comment:
Nit: I think we can use `Date.now()` to get the same effect as creating, stringifying, and parsing.

Reply:
@sri-byte I don't know the analytics code very well, but do we need to explicitly include timestamps? I would have thought the timestamp was already included in the event data that is being sent. I do see the ticket calls for timestamp, but I would have thought that data is already accessible without adding it explicitly here.

Reply:
We don't need the timestamp @neallred

Reply:
@AGMETEOR based on Sri's comment, recommend removing `utcTimeStamp` in the various places it is passed.